src/plugins/data/public/query/query_service.ts

```diff
@@ -80,8 +80,8 @@ export class QueryService {
       savedQueries: createSavedQueryService(savedObjectsClient),
       state$: this.state$,
       timefilter: this.timefilter,
-      getEsQuery: (indexPattern: IndexPattern) => {
-        const timeFilter = this.timefilter.timefilter.createFilter(indexPattern);
+      getEsQuery: (indexPattern: IndexPattern, timeRange?: TimeRange) => {
```

Comment:
Added additional parameter (`createFilter` already had 2nd optional argument). 
It was convenient to use this in an example